### PR TITLE
Dsl2

### PR DESCRIPTION
**Pull request by aunderwo**
This commit adds the ability to handle references with multiple chromosomes
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/bactmap/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/bactmap _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -154,6 +154,30 @@ Output files
...
+
+Statistics gathered when trimming reads
+![FastP Statistics](images/multiqc_fastp.png)
+
+#### Mapping Statistics
+
+Statistics gathered when mapping reads
+![Mapping Statistics](images/multiqc_mapping_stats.png)
+
+#### Varinat Statistics
```
Comment (ewels) [inline docs/output.md:176]:
```suggestion
#### Variant Statistics
```

File: docs/output.md [hunk 2]
```diff
@@ -154,6 +154,30 @@ Output files
   * `output.raxml.bestTree` A ML tree built with RAxML-NG selected as the best after running ML
   * `output.raxml.support` A ML tree built with RAxML-NG with support values for branches based on bootstrapping
 
+### MultiQC
+
+Various quality statistics are compiled from the previous outputs using the [MultiQC](https://multiqc.info/) software:
```
Comment (ewels) [inline docs/output.md:159]: It's nice if you can give the file paths for output files, as done above

**Review by ewels — APPROVED** [review]
LGTM from a speed read through 👍🏻

**Review by maxulysse — APPROVED** [review]
LGTM

**Comment by aunderwo** [comment]
Thanks @maxulysse for the approval

**Pull request merged**